apps/framework-docs/src/pages/getting-started/version-status.mdx

```diff
@@ -1,31 +1,30 @@
-
 # Moose is Alpha Software
-Thank you so much for your interest in [Moose](https://www.moosejs.com/), the developer framework for your data & analytics stack. Moose is still in Alpha, and is not yet recommended for critical production use cases. That being said, we’d love for you to get your hands on it and try it out.
+
+Thank you so much for your interest in [Moose](https://www.moosejs.com/), the developer framework for your data & analytics stack. Moose is still in Alpha, and is not yet recommended for critical production use cases. That said, we’d love for you to get your hands on it and try it out.
 
 Moose is in active development, and we’re looking for feedback and insights on how to harden and improve, as we build towards the Beta release. You can reach us at [hello@moosejs.dev](mailto:hello@moosejs.dev) or in our [developer communities](https://www.moosejs.com/community) on [Slack](https://join.slack.com/t/moose-community/shared_invite/zt-2fjh5n3wz-cnOmM9Xe9DYAgQrNu8xKxg) and [Github](https://github.com/514-labs/moose).
 
 # Versioning
 
-All packages and builds are synced in version when published to repositories. It makes it easy to understand the compatibility levels in the different components.
+All packages and builds are synced and versioned when published to repositories. It makes it easy to understand the compatibility levels in the different components.
```

Comment:
All packages and builds are synced `in version` when < didn't make sense to me.  So I changed that to `and versioned` since that made some sense to me.